src/html_generator.py

```diff
@@ -0,0 +1,93 @@
+import tmt
+from tmt import Test, Logger, Plan
+
+
+def generate_test_html_page(test: Test, logger: Logger) -> str:
+    """
+    This function generates an HTML file with the input data for a test
+    :param test: Test object
+    :param logger: tmt.Logger instance
+    :return:
+    """
+    logger.print("Generating the HTML file...")
+    full_url = test.web_link()
+    # Adding the input data to the HTML file
+    file_html = ('''<html>
+    <head>
+    <title>HTML File</title>
+    <meta charset="UTF-8">
+    </head> 
+    <body>
+    name: ''' + str(test.name) + '''<br>
```

Comment:
```suggestion
    name: {test.name)<br>
```

It makes the code much cleaner/shorter.